cloudinit/config/cloud-init-schema.json

```diff
@@ -838,13 +838,13 @@
               "minItems": 1,
               "maxItems": 6
           },
-          "description": "List of lists. Each inner list entry is a list of ``/etc/fstab`` entries of the format: [ fs_spec, fs_file, fs_vfstype, fs_mntops, fs-freq, fs_passno ]",
+          "description": "List of lists. Each inner list entry is a list of ``/etc/fstab`` mount declaratiosn of the format: [ fs_spec, fs_file, fs_vfstype, fs_mntops, fs-freq, fs_passno ]. A mount declaration with less than 6 items, will get remaining values from ``mount_default_fields``. A mount declaration with only `fs_spec` and no `fs_file` mountpoint will be skipped.",
```

Comment:
`declaratiosn`->`declarations`. Also the comma in the second sentence should be removed.